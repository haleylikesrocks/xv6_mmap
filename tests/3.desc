Anonymous mmap :Testing Anonymous mmap without PROT_WRITE flag
