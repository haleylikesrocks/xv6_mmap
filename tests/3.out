XV6_TEST_OUTPUT : mmap good
XV6_TEST_OUTPUT : Strlen Before modification: 0
